nautobot/core/tests/test_settings_schema.py

```diff
@@ -0,0 +1,208 @@
+import os
+import sys
+from unittest import mock, TestCase
+
+from django.conf import global_settings as django_defaults
+from jsonschema.exceptions import SchemaError, ValidationError
+from jsonschema.validators import Draft202012Validator
+import yaml
+
+SETTINGS_DOCUMENTATION_SCHEMA = {
+    "$id": "https://nautobot.com/2.2/settings-schema/strict",
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$vocabulary": {
+        "https://json-schema.org/draft/2020-12/vocab/core": True,
+        "https://json-schema.org/draft/2020-12/vocab/applicator": True,
+        "https://json-schema.org/draft/2020-12/vocab/unevaluated": True,
+        "https://json-schema.org/draft/2020-12/vocab/validation": True,
+        "https://json-schema.org/draft/2020-12/vocab/meta-data": True,
+        "https://json-schema.org/draft/2020-12/vocab/format-annotation": True,
+        "https://json-schema.org/draft/2020-12/vocab/content": True,
+    },
+    "$dynamicAnchor": "meta",
+    "$ref": "https://json-schema.org/draft/2020-12/schema",
+    "properties": {
+        "default_literal": {
+            "type": "string",
+        },
+        "details": {
+            "type": "string",
+        },
+        "environment_variable": {
+            "type": "string",
+        },
+        "is_constance_config": {
+            "type": "boolean",
+        },
+        "is_required_setting": {
+            "type": "boolean",
+        },
+        "see_also": {
+            "type": "object",
+        },
+        "version_added": {
+            "type": "string",
+        },
+    },
+    "unevaluatedProperties": False,
+}
+
+
+class SettingsJSONSchemaTestCase(TestCase):
```

Comment:
Don't forget to manually tag this testcase since it's not one of our base testcases.